<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestioneAnagrafica</class>
 <widget class="QMainWindow" name="GestioneAnagrafica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestione Anagrafica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpInserisciCittadino">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Inserisci o Ricerca Cittadino</string>
    </property>
    <widget class="QPushButton" name="BtnInserisci">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Inserisci</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtVia">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCognome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cognome</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNome">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVia">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Via/Viale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCivico">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCivico">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>N°</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCognome">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNome">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblRagioneSociale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ragione sociale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRagioneSociale">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBtnPrivato">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Privato</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBtnAzienda">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Azienda</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnCerca">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cerca</string>
     </property>
    </widget>
    <zorder>BtnInserisci</zorder>
    <zorder>txtVia</zorder>
    <zorder>lblCognome</zorder>
    <zorder>lblNome</zorder>
    <zorder>lblVia</zorder>
    <zorder>txtCivico</zorder>
    <zorder>lblCivico</zorder>
    <zorder>txtCognome</zorder>
    <zorder>lblRagioneSociale</zorder>
    <zorder>txtRagioneSociale</zorder>
    <zorder>rBtnPrivato</zorder>
    <zorder>rBtnAzienda</zorder>
    <zorder>txtNome</zorder>
    <zorder>BtnCerca</zorder>
   </widget>
   <widget class="QTableView" name="tblRicerca">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>611</width>
      <height>111</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QCommandLinkButton" name="cmdLinkBtnElimina">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Elimina cittadino</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="cmdLinkBtnModifica">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifica cittadino</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpModificaCittadino">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>611</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Modifica Cittadino</string>
    </property>
    <widget class="QLineEdit" name="txtViaMod">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCognomeMod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cognome</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNomeMod">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblViaMod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Via/Viale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCivicoMod">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCivicoMod">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>N°</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCognomeMod">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNomeMod">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblRagioneSocialeMod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ragione sociale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRagioneSocialeMod">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnModifica">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifica</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnAnagraficaEsci">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Esci</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
